react/LocaleSwitcher.js

```diff
@@ -1,3 +1,100 @@
-import LocaleSwitcher from './components/index'
+import React, { useState, useEffect, useMemo } from 'react'
+import { graphql } from 'react-apollo'
+import { useRuntime } from 'vtex.render-runtime'
+import { IconGlobe } from 'vtex.store-icons'
+import Locales from './queries/locales.gql'
+import { useCssHandles } from 'vtex.css-handles'
 
-export default LocaleSwitcher
\ No newline at end of file
+const CSS_HANDLES = ['container', 'button', 'buttonText', 'list', 'listElement']
```

Comment:
love this change of moving the code for just one file

Reply:
Me too 😍